### ISSUE TITLE
Don't keep track of Pins with remote status

### ISSUE THREAD

**Issue by vasco-santos** [web3-storage/web3.storage#1098]
We currently keep all the Pins, even when remote status. This does not scale as number of peers in cluster gets bigger

**Comment by alanshaw** [comment]
Often when calling "status" immediately after adding the statuses will ALL be "Unpinned" so it won't work for most uploads.

I seem to remember hector telling me that the pin allocations actually come back in the response from a call to `/add` but I don't think that's the case. I will ask and update here.

**Comment by alanshaw** [comment]
Hector wants to fix this, I've opened an issue to track https://github.com/ipfs/ipfs-cluster/issues/1598

**Comment by dchoi27** [comment]
can we close this?

**Comment by flea89** [comment]
I've picked this up given we have a few hundred millions "Remote" Pins in prod that should be cleaned up.

Deleting the Pins should sensibly improve performance across the board and especially help with storage notification job.

**Comment by flea89** [comment]
Follow-up PR #1619 to update db type once this is merged, release, and cron have been run.
